src/dll/Systems/PluginSystem.hpp

```diff
@@ -25,6 +25,12 @@ class PluginSystem : public ISystem
     using Map_t = std::unordered_map<HMODULE, std::shared_ptr<PluginBase>>;
     using MapIter_t = Map_t::iterator;
 
+    struct PluginEntry
```

Comment:
I think this name can be a bit confusing if you do not look at the context of where it is used. I would suggest `PluginLoadInfo`, what do you think?

Reply:
I was not convinced myself, I'll update it soon.